global.json

```diff
@@ -8,6 +8,6 @@
     "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.25161.4"
   },
   "sdk": {
-    "allowPrerelease": true
+    "allowPrerelease": false
```

Comment:
This seems more like a change doing some tests than a necessary change, right?

Reply:
Nop this was me to fix some issues we have now when net10 preview is installed